Platforms/QemuQ35Pkg/QemuQ35Pkg.fdf

```diff
@@ -559,6 +559,7 @@ FILE FREEFORM = PCD(gZeroTouchPkgTokenSpaceGuid.PcdZeroTouchCertificateFile) {
   INF QemuPkg/FrontPageButtons/FrontPageButtons.inf
   INF OemPkg/FrontPage/FrontPage.inf
   INF PcBdsPkg/MsBootPolicy/MsBootPolicy.inf
+  INF MsCorePkg/LoadOptionVariablePolicyDxe/LoadOptionVariablePolicyDxe.inf

```

Comment:
This is ultimately fine, but if you could add this a little closer to `MuVarPolicyFoundationDxe.inf`, it will potentially bring the driver dispatch forward in boot and allow policies to be applied sooner.

Reply:
Moved to just after MuVarPolicyFoundationDxe